index.html

```diff
@@ -602,6 +602,9 @@ <h2 slot=header>
           <a href=/examples/input-output.html>&lt;input-output&gt;</a>
 
         <li>
+          <a href=/examples/to-do>&lt;to-do&gt;</a>
```

Comment:
Correction. Looks like since 1999 DOM Level 0 @tmornini. Article came out in 2008 and author stated _"9 years later the information still isn't out there"_ https://meiert.com/en/blog/optional-tags-in-html-4/